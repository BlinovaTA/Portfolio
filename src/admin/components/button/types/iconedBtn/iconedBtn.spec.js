import iconedBtn from "./iconedBtn.vue";
import { mount } from "@vue/test-utils";

it("iconed button snapshot", async () => {
  const mountWrapper = mount(iconedBtn);
  expect(mountWrapper.element).toMatchSnapshot();
});