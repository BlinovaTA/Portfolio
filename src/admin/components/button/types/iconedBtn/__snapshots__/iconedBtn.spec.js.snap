// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`iconed button snapshot 1`] = `
<button
  class="button-component"
>
  Добавить группу
</button>
`;
